tests/rkt_trust_test.go

```diff
@@ -78,6 +83,21 @@ func TestTrust(t *testing.T) {
 	runImage(t, ctx, imageFile, "Hello", false)
 	runImage(t, ctx, imageFile2, "openpgp: signature made by unknown entity", true)
 
+	t.Logf("Skip trusted key (rkt trust --skip-trusted) with trusted key absent\n")
+	runRktTrustSkipTrustedTrue(t, ctx, "rkt-skip-trusted.com/my-app", 1, false)
+
+	t.Logf("Skip trusted key (rkt trust --skip-trusted) with trusted key present\n")
+	runRktTrustSkipTrustedTrue(t, ctx, "rkt-skip-trusted.com/my-app", 1, true)
+
+	t.Logf("Don't skip trusted key (rkt trust --skip-trusted=false) with trusted key present\n")
+	runRktTrustSkipTrustedFalse(t, ctx, "rkt-skip-trusted.com/my-app", 1, true)
+
+	t.Logf("Don't skip trusted key (rkt trust --skip-trusted=false) with trusted key absent\n")
+	runRktTrustSkipTrustedFalse(t, ctx, "rkt-skip-trusted.com/my-app", 1, false)
+
+	t.Logf("Now the image can be executed\n")
+	runImage(t, ctx, imageFile3, "Hello", false)
```

Comment:
Could you also try to run the image with `runImage` after each runRktTrustSkipTrusted* and check it has the correct behavior?